src/Platforms/PostgreSQLPlatform.php

```diff
@@ -809,6 +809,22 @@ public function getDropForeignKeySQL($foreignKey, $table)
         return $this->getDropConstraintSQL($foreignKey, $table);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getDropIndexSQL($index, $table = null)
+    {
+        if ($index instanceof Index && $index->isPrimary()) {
+            $constraintName = 'primary' === $index->getName() ? $table . '_pkey' : $index->getName();
+            return $this->getDropConstraintSQL($constraintName, $table);
+        } elseif (is_string($index) && '"primary"' === $index) {
```

Comment:
There is no need for an `else` condition when the previous one breaks the execution flow with a `return` statement.
Additionally, I think there is no need for the `is_string()` check here.